### PR TITLE
Di 20910 dimx relations

### PR DESCRIPTION
**Pull request by saargt**

### CODE REVIEW

**Comment by ShirMarciano** [comment]
@saargt  - Should you push the batch changes to this branch before I review it??
also - the unit test fail - need to check?

File: server-side/data_source_api.ts [hunk 1]
```diff
@@ -26,11 +26,8 @@ export async function purge(client: Client, request: Request)
 	switch (request.method) 
 	{
 	case "POST": {
-		const papiClient = Helper.getPapiClient(client);
-		const papiService = new PapiService(papiClient);
-		const core = new CoreSchemaService(request.body.Name, request, papiService);
-
-		return await core.purgeSchema();
+		const coreSchema = getCoreSchemaService(client, request);
```
Comment (ShirMarciano) [inline server-side/data_source_api.ts:29]: 👍🏼

File: server-side/installation.ts [hunk 2]
```diff
@@ -9,23 +9,91 @@ The error Message is importent! it will be written in the audit log and help the
...
+	const isHidden = false;
+	const { importRelation, exportRelation }: { importRelation: Relation; exportRelation: Relation; } = getDimxRelationsBodies(client, isHidden);
+	
+	await upsertRelation(papiClient, importRelation);
+	await upsertRelation(papiClient, exportRelation);
+}
+
+async function removeDimxRelations(papiClient: PapiClient, client: Client) 
+{
+	const isHidden = true;
```
Comment (ShirMarciano) [inline server-side/installation.ts:66]: This is the only difference between the createDimxRelation and RemoveDimxRelation code?

Comment (saargt) [reply server-side/installation.ts:63]: Done @ShirMarciano

**Pull request merged**